<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Pranith-PC" name="Command line test" tests="6" failures="1" timestamp="6 Jan 2020 07:24:15 GMT" time="31.62" errors="0">
  <testcase name="verifylogin" time="0.067" classname="com.app.tests.DressPageTest">
    <skipped/>
  </testcase> <!-- verifylogin -->
  <testcase name="verifydisplayalltab" time="0.952" classname="com.app.tests.HomePageTest"/>
  <testcase name="verifydressestab" time="4.403" classname="com.app.tests.HomePageTest"/>
  <testcase name="verifyemailtab" time="18.802" classname="com.app.tests.HomePageTest">
    <failure type="java.lang.AssertionError" message="Failed expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Failed expected [true] but found [false]
	at com.app.tests.HomePageTest.verifyemailtab(HomePageTest.java:52)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:77)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.executeMulti(TestNGDirectoryTestSuite.java:159)
	at org.apache.maven.surefire.testng.TestNGDirectoryTestSuite.execute(TestNGDirectoryTestSuite.java:99)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:106)
	at org.apache.maven.surefire.util.ReflectionUtils.invokeMethodWithArray(ReflectionUtils.java:189)
	at org.apache.maven.surefire.booter.ProviderFactory$ProviderProxy.invoke(ProviderFactory.java:165)
	at org.apache.maven.surefire.booter.ProviderFactory.invokeProvider(ProviderFactory.java:85)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:115)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:75)
... Removed 28 stack frames]]>
    </failure>
  </testcase> <!-- verifyemailtab -->
  <testcase name="verifytshirttab" time="3.06" classname="com.app.tests.HomePageTest"/>
  <testcase name="verifywomentab" time="4.128" classname="com.app.tests.HomePageTest"/>
</testsuite> <!-- Command line test -->
